src/routes/webhooks/paddle.ts

```diff
@@ -166,6 +166,115 @@ const logPaddleAnalyticsEvent = async (
   ]);
 };
 
+const concatText = (a: string, b: string) => `${a}\n${b}`;
```

Comment:
Something like this?

```suggestion
const concatText = (a: string, b: string) => [a, b].filter(Boolean).join(`\n`)
```

to avoid concating empty values